Serialization/SerializeToJson.java

```diff
@@ -0,0 +1,63 @@
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SerializeToJson {
+    private static StringBuilder sb = new StringBuilder();
+    private static int count = 0;
+    private static String tabRepeat = String.join("", Collections.nCopies(count, "\t"));
+    private static List<Group> childGroups = new ArrayList();
+    private static List<Figure> list = new ArrayList();
+
+    public static void serialize(Group group) {
```

Comment:
you probably want this method to return string. Or it should write into file. 

Printing into console typically isn't super useful.